### DISCUSSION TITLE
Welcome to hellpanderrr.github.io Discussions!

### DISCUSSION THREAD

**Discussion by hellpanderrr** [hellpanderrr/hellpanderrr.github.io#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Akkatosh** [comment]
Hello. First of all - thank you very much for this converter, it is one of the best, at least of those I know.
Secondly - can we expect to add other languages? I don't suggest English, but how about Irish?

**Reply by hellpanderrr** [reply]
Hi. Thanks for the feedback. I add languages which have relatively mature modules (with tests) on Wiktionary https://en.wiktionary.org/wiki/Category:Pronunciation_modules Irish does not have one; I am looking into whether its possible to create a similiar script from scratch based on _The Sound Structure of Modern Irish_ by Raymond Hickey, so maybe I'll be able to add it in the future.

**Reply by hellpanderrr** [reply]
@Akkatosh Added an attempt at Irish module here https://hellpanderrr.github.io/wiktionary_pron/?lang=Irish 
https://github.com/hellpanderrr/IrishPhoneticTranscription

**Comment by Maxyjjjjj** [comment]
Hello.

First of all — thank you VERY much for this tool, it is the best IPA converter I know of

Secondly — any plans for Mongolian and/or Nivkh?, you can look for other than wiktionary, and if it's hard to find, you may make own stuff, if you can

**Reply by hellpanderrr** [reply]
Hi there. There's actually a wiktionary module for Mongolian, it's used for generating transcriptions you see on wiktionary e.g. https://en.wiktionary.org/wiki/%D0%B0%D0%B1%D1%81%D1%82%D1%80%D0%B0%D0%BA%D1%82%D0%B0%D0%B0%D1%80 Are they good enough?

**Reply by Maxyjjjjj** [reply]
yup, i guess
